packages/web/src/components/Layout/MarketingLayout.tsx

```diff
@@ -12,9 +11,9 @@ export const MarketingLayout: React.FC = ({ children }) => {
         <GiCrossFlare size="36px" />
         <Heading>Project X</Heading>
         <Spacer />
-        <Link href="/app" passHref>
+        <Link href="/api/auth/github/login">
           <Button as="a" size="sm" colorScheme="blue">
```

Comment:
You don't need the link because the button is acting as an `a` tag with the `as="a"` property. You can just add the href to the button

```suggestion
          <Button as="a" href="/api/auth/github/login" size="sm" colorScheme="blue">
```